packages/ui/src/views/layouts/SandboxLayout.tsx

```diff
@@ -44,7 +44,7 @@ function Header({ children, className }: { children: ReactNode; className?: stri
   return (
     <header
       className={cn(
-        'bg-cn-background fixed translate-x-[--cn-sidebar-width] transition-transform duration-200 ease-linear right-0 top-0 z-40 h-[var(--cn-breadcrumbs-height)]',
+        'bg-cn-background fixed transition-transform duration-200 ease-linear right-0 top-0 z-40 h-[var(--cn-breadcrumbs-height)]',
```

Comment:
can we fix `h-[var(--cn-breadcrumbs-height)]` as well?

Reply:
This is a utility variable that helps manage different cases across several components in the app. It is added in the `packages/ui/src/styles/styles.css`
I wouldn't start refactoring it in this PR - it can take a noticeable amount of time.

<img width="1373" height="474" alt="image" src="https://github.com/user-attachments/assets/d5fd5d60-a45b-47ce-9bb1-1adea1d7bd32" />